src/Avalonia.ReactiveUI/TransitioningContentControl.cs

```diff
@@ -57,7 +57,7 @@ public class TransitioningContentControl : ContentControl, IStyleable
         
         /// <summary>
         /// TransitioningContentControl uses the default ContentControl 
-        /// template from Avalonia default theme.
+        /// template from Avalonia simple theme.
```

Comment:
```suggestion
        /// template from Avalonia default theme.
```
It's theme independed.

Reply:
Ah, ok, I guess I should have read the code a little more here -- it was just a find and replace.